components/closebutton/index.css

```diff
@@ -231,55 +245,6 @@ a.spectrum-CloseButton {
   }
 }
 
-/* Modifier Classes */
-.spectrum-CloseButton--staticBlack:not(:disabled),
-.spectrum-CloseButton--staticWhite:not(:disabled) {
```

Comment:
I don't think we need these? We should be able to redefine the token values [like I've done here](https://github.com/adobe/spectrum-css/pull/2564/files#diff-a4409bdab1797d61664b594472edb087eeb1c2c2ce52c0c14b68dc4b5fe574c7R75-R109) and just rely on that for the static background updates.

Reply:
Nice refactor that makes better use of CSS variables 👍